Integrations/integration-FreshDesk.yml

```diff
@@ -0,0 +1,2574 @@
+    - {contextPath: FD.Ticket.Conversation.UpdatedAt, description: Conversation updated
+        timestamp, type: Date}
+    - {contextPath: FD.Ticket.Conversation.FromEmail, description: The email address
+        from which the reply/note is sent. By default the global support email will
+        be used., type: String}
+  - arguments:
+    - default: false
+      defaultValue: ''
+      description: ID of the ticket to delete
+      isArray: false
+      name: id
+      predefined: []
+      required: true
+    description: Delete the ticket specified by the 'id' command argument
+    execution: false
+    name: fd-delete-ticket
+    outputs: []
+  - arguments:
+    - default: false
+      defaultValue: ''
+      description: Predefined filters
+      isArray: false
+      name: filter
+      predefined: [new_and_my_open, watching, spam, deleted]
+      required: false
+    - default: false
+      defaultValue: ''
+      description: Filter by either the ticket requester's email or ID
+      isArray: false
+      name: requester
+      predefined: []
+      required: false
+    - default: false
+      defaultValue: ''
+      description: Filter by tickets associated with the specified company ID
+      isArray: false
+      name: company_id
+      predefined: []
+      required: false
+    - default: false
+      defaultValue: ''
+      description: By default, only tickets that have been created within the past
+        30 days will be returned. For older tickets, use this filter. Example value
+        for this field would be "2015-01-19T02:00:00Z"
+      isArray: false
+      name: updated_since
+      predefined: []
+      required: false
+    - default: false
+      defaultValue: ''
+      description: Reference field for ordering the list of tickets. The default sort
+        order is created_at.
+      isArray: false
+      name: order_by
+      predefined: [created_at, due_by, updated_at, status]
+      required: false
+    - default: false
+      defaultValue: ''
+      description: Whether to order the resulting tickets in ascending or descending
+        order. The default is descending.
+      isArray: false
+      name: order_type
+      predefined: [asc, desc]
+      required: false
+    - default: false
+      defaultValue: ''
+      description: If set to 'yes' then ticket’s closed_at, resolved_at and first_responded_at
+        time will be included.
+      isArray: false
+      name: include_stats
+      predefined: ['yes', 'no']
+      required: false
+    - default: false
+      defaultValue: ''
+      description: If set to 'yes' then the ticket requester's id, email, mobile,
+        name, and phone will be included in the ticket's output for each ticket.
+      isArray: false
+      name: include_requester
+      predefined: ['yes', 'no']
+      required: false
+    - default: false
+      defaultValue: ''
+      description: If set to 'yes' then the ticket's description and description_text
```

Comment:
Yea I think we do need it, because when listing a bunch of tickets, embedding additional information counts as additional API calls against your quota.
> 4. Use 'include' to embed additional details in the response. Each include will consume an additional 2 credits. For example if you embed the stats information you will be charged a total of 3 API credits for the call.

Reply:
please mention the quota in the description